dhcp/unikernel.ml

```diff
@@ -0,0 +1,85 @@
+open V1_LWT
+open Lwt
+
+(* IP Configuration, all you need besides dhcpd.conf. *)
+let ipaddr = Ipaddr.V4.of_string_exn "192.168.1.5"
```

Comment:
> If there are things in (eg) ethif example that needed updating for this example, then possible skeleton needs some general TLC again to bring it back up to date.

I did that when in my functoria PR.

Also, now, mirage's (and functoria's) travis run mirage-skeleton.